panoptes_client/panoptes.py

```diff
@@ -58,6 +58,7 @@ class Panoptes(object):
         'GET': {},
         'PUT': {
             'Content-Type': 'application/json',
+            'x-ms-blob-type': 'BlockBlob',
```

Comment:
this is not where the media uploads to the cloud object store happen, rather this is the default request headers for interacting with the panoptes API

Reply:
oops